lib/itamae/resource/file.rb

```diff
@@ -20,12 +20,16 @@ def pre_action
         when :edit
           attributes.exist = true
 
-          content = backend.receive_file(attributes.path)
-          attributes.block.call(content)
-          attributes.content = content
+          unless options[:dry_run]
+            content = backend.receive_file(attributes.path)
+            attributes.block.call(content)
+            attributes.content = content
+          end
         end
 
-        send_tempfile
+        unless options[:dry_run]
```

Comment:
`create` action in dry-run mode shows difference between tempfile and current file. This break the ability.